ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -268,6 +283,11 @@ RnsShell::LayerInvalidateMask  RSkComponentTextInput::updateComponentProps(const
     }
   }
 
+  if (maxLength_ != textInputProps.maxLength) {
+    maxLength_ = textInputProps.maxLength;
+    mask |= LayerPaintInvalidate;
+  }
```

Comment:
we may not need paint invalidate here for this prop right ? since this prop is applicable when user types keys. so we can directly assign the value .